commons/src/main/java/org/infinispan/commons/jmx/JmxUtil.java

```diff
@@ -78,7 +78,7 @@ public static void registerMBean(Object mbean, ObjectName objectName, MBeanServe
     * @throws Exception If unregistration could not be completed.
     */
    public static void unregisterMBean(ObjectName objectName, MBeanServer mBeanServer) throws Exception {
-      if (mBeanServer.isRegistered(objectName)) {
+      if (objectName != null && mBeanServer.isRegistered(objectName)) {
```

Comment:
IMO the caller should do this check, in fact most callers already do.

Reply:
Ok, will remove it here and add it where it's missing in the callers.